ios/keyman/Keyman/Keyman.xcodeproj/project.pbxproj

```diff
@@ -1266,13 +1288,12 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				INFOPLIST_FILE = SWKeyboard/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
-				KEYMAN_ROOT = "$(SRCROOT)/../../..";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @executable_path/../../Frameworks";
 				LIBRARY_SEARCH_PATHS = "$(inherited)";
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = NO;
 				OTHER_LDFLAGS = "";
-				PRODUCT_BUNDLE_IDENTIFIER = Tavultesoft.Keyman.SWKeyboard;
+				PRODUCT_BUNDLE_IDENTIFIER = Tavultesoft.Keyman.dev.SWKeyboard;
```

Comment:
This tag must be replicated for app-extensions due to Apple requirements on bundle identifiers, and it'll be pretty useful once we get errors out of the system keyboard, too.

Reply:
... and note that it gets immediately removed in the child PR #2782.  A better implementation was realized during its development.

Reply:
Eek. Hard for reviewers to keep up with this!

Reply:
Yeah.  It required a number of extra tweaks to do it the "better" way, so I decided against back-porting it to this PR, which would have caused some "bloat" here.